README.md

```diff
@@ -28,27 +28,31 @@ module:
 
 ## How to compile AddOn code
 
-### C code/Scripts, with GRASS source code on your computer
+### C code/Scripts, with GRASS GIS core source code on your computer
 
-Preparations (assuming the source code in $HOME/grass/):
-(if you have already built GRASS from source you don't need to do this
-again. If adding to a binary install, the versions must match exactly.
-You need to git checkout the exact tag or commit used for the binary.)
+Preparations (assuming the [GRASS GIS core source code](https://github.com/OSGeo/grass)
+being stored in `$HOME/grass/` - if you have already built GRASS GIS core from
+source code you don't need to do this again. If adding to a binary install,
+the versions must match exactly. You need to `git checkout` the exact tag
+or commit used for the binary.)
 
 ```
+# GRASS GIS core source code
 ./configure # [optionally flags]
 make libs
```

Comment:
libs does not makes sense when this is meant for Python scripts as well. Simple make is appropriate.